doc/api/esm.md

```diff
@@ -582,6 +608,9 @@ node --experimental-wasm-modules index.mjs
 
 would provide the exports interface for the instantiation of `module.wasm`.
 
+The `assert { type: 'webassembly' }` syntax is mandatory; see
```

Comment:
should we link out to the ongoing discussion in HTML spec?

Reply:
I’m not sure what the best link would be. There’s https://github.com/WebAssembly/esm-integration/issues/42 or https://github.com/tc39/proposal-import-assertions/issues/19 or others. It also feels a little messy to link to a discussion thread as opposed to a decision outcome. But if you have a link in particular you recommend, though, I’m happy to add it.